package/MDAnalysis/coordinates/base.py

```diff
@@ -141,9 +141,15 @@
 from ..auxiliary.core import get_auxreader_for
 
 class Namespace(object):
-    """Empty class 
-    To allow storing attributes in new namespace """
-    pass
+    """Class to allow storing attributes in new namespace. """
```

Comment:
This doesn't then allow for tab completion which is annoying... because a dict doesn't have `__dict__`.  But otherwise I'd like all the other `dict` stuff like keys, iteritems etc